.fauna/credentials/access_keys

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I'm not sure we actually want/need this in source control. Did they sneak in after .fauna was removed from gitignore?

Reply:
oh yikes that shouldn't be in there. ran tests with wrong `homedir` mock